hyperion-geiger-counter/src/main/java/com/willowtreeapps/hyperion/geigercounter/DroppedFrameObserver.java

```diff
@@ -0,0 +1,109 @@
+package com.willowtreeapps.hyperion.geigercounter;
+
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.content.res.AssetFileDescriptor;
+import android.media.SoundPool;
+import android.util.Log;
+import android.view.Choreographer;
+
+import static android.media.AudioManager.STREAM_SYSTEM;
+import static com.willowtreeapps.hyperion.geigercounter.GeigerCounterPlugin.LOG_TAG;
+
+@SuppressLint("NewApi")
+public class DroppedFrameObserver implements Choreographer.FrameCallback {
```

Comment:
Naming here could be a little more clear. `DroppedFrameCallback` maybe since it's a `FrameCallback` and not an `Observer`.

Reply:
Read further on and I see where you're going.

Reply:
Yeah, the fact that it is a FrameCallback is an implementation detail that consumers don't need to think about. The public API is just, create this thing that watches for dropped frames, and switch it on and off.

Is there a better name than Observer for this that doesn't collide with Android terms?